### PR TITLE
Louis/update ansible

### PR DESCRIPTION
**Pull request by LinuxGit**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@lilin90 PTAL

File: op-guide/ansible-deployment.md [hunk 1]
```diff
@@ -26,15 +26,17 @@ Ansible 是一款自动化运维工具，[TiDB-Ansible](https://github.com/pingc
     - 依赖 Python Jinja2 及 MarkupSafe 指定版本模块: `pip install Jinja2==2.7.2 MarkupSafe==0.11`
     - 可通过 ssh 登录目标机器，支持密码登录或 ssh authorized_key 登录。
     - 中控机可以是部署目标机器中的某一台，该机器需开放外网访问，并且安装 curl 软件包，用于下载 binary。
+    - 如使用 Docker 方式部署，依赖详见[如何使用 docker 方式部署 TiDB](https://github.com/pingcap/docs-cn/blob/master/op-guide/ansible-deployment.md#如何使用docker方式部署TiDB)， 默认为 binary 部署方式。
```
Comment (lilin90) [inline op-guide/ansible-deployment.md]: Delete the extra space before "默认".

Comment (LinuxGit) [reply op-guide/ansible-deployment.md]: Done

File: op-guide/ansible-deployment.md [hunk 2]
```diff
@@ -26,15 +26,17 @@ Ansible 是一款自动化运维工具，[TiDB-Ansible](https://github.com/pingc
...
 
 2.  部署目标机器若干
 
     - 建议4台及以上，TiKV 至少3实例，且与 TiDB、PD 模块不位于同一主机,详见[部署建议](recommendation.md)。
     - Linux 操作系统，x86_64 架构(amd64)，内核版本建议 3.10 以上，推荐 CentOS 7.3 及以上版本, 文件系统推荐 ext4(部分内核版本 xfs 文件系统有 bug, 本工具检查到 xfs 文件系统有 bug 会退出)。
     - 机器之间网络互通，防火墙、iptables 等可以在部署验证时关闭，后期开启。
-    - 机器的时间、时区设置正确(要求机器时间同步)，有 NTP 服务可以同步正确时间, ubuntu 系统需单独安装 ntpstat 软件包。
+    - 机器的时间、时区设置正确(要求机器时间同步)，有 NTP 服务可以同步正确时间, ubuntu 系统需单独安装 ntpstat 软件包, 详见[如何检测 NTP 服务是否正常](https://github.com/pingcap/docs-cn/blob/master/op-guide/ansible-deployment.md#如何检测NTP服务是否正常)。
```
Comment (lilin90) [inline op-guide/ansible-deployment.md]: Pls update the format of two commas.

File: op-guide/ansible-deployment.md [hunk 3]
```diff
@@ -26,15 +26,17 @@ Ansible 是一款自动化运维工具，[TiDB-Ansible](https://github.com/pingc
...
     - 若使用普通用户作为 Ansible SSH 远程连接用户，该用户需要有 sudo 到 root 权限，或直接使用 root 用户远程连接。
     - Python 2.6 或 Python 2.7。
+    - 如使用 Docker 方式部署，依赖详见[如何使用 docker 方式部署 TiDB](https://github.com/pingcap/docs-cn/blob/master/op-guide/ansible-deployment.md#如何使用docker方式部署TiDB), 默认为 binary 部署方式。
```
Comment (lilin90) [inline op-guide/ansible-deployment.md]: Comma format.

File: op-guide/ansible-deployment.md [hunk 4]
```diff
@@ -411,3 +453,63 @@ git clone -b release-1.0 https://github.com/pingcap/tidb-ansible.git
...
+Unable to talk to NTP daemon. Is it running?
+```
+使用以下命令可使 NTP 服务尽快开始同步，pool.ntp.org 可替换为您的 NTP server：
+```
+$ sudo systemctl stop ntpd.service
+$ sudo ntpdate pool.ntp.org
+$ sudo systemctl start ntpd.service
+```
+
+### 如何使用docker方式部署TiDB
```
Comment (lilin90) [inline op-guide/ansible-deployment.md]: Add space between English and Chinese characters.

File: op-guide/ansible-deployment.md [hunk 5]
```diff
@@ -357,28 +392,33 @@ location_labels = ["host"]
...
+|滚动升级监控组件|`ansible-playbook rolling_update_monitor.yml`|
 
 > TiDB 服务数据迁移、性能调优等更多高级功能请参考 [https://github.com/pingcap/docs-cn](https://github.com/pingcap/docs-cn)
 
 ## 常见部署问题
 
 ### TiDB 各版本下载链接
+- master 版本：
+    - [TiDB master-CentOS7](http://download.pingcap.org/tidb-latest-linux-amd64-unportable.tar.gz)
+    - [TiDB m aster-CentOS6](http://download.pingcap.org/tidb-latest-linux-amd64-unportable-centos6.tar.gz)
```
Comment (lilin90) [inline op-guide/ansible-deployment.md]: "TiDB m aster" -> "TiDB master"

File: op-guide/ansible-deployment.md [hunk 6]
```diff
@@ -411,3 +453,63 @@ git clone -b release-1.0 https://github.com/pingcap/tidb-ansible.git
...
+```
+$ ntpstat
+unsynchronised
+```
+以下情况表示 NTP 服务未正常运行：
+```
+$ ntpstat
+Unable to talk to NTP daemon. Is it running?
+```
+使用以下命令可使 NTP 服务尽快开始同步，pool.ntp.org 可替换为您的 NTP server：
```
Comment (lilin90) [inline op-guide/ansible-deployment.md]: Delete "您的".

**Pull request merged**